soyjs/exec.go

```diff
@@ -25,15 +25,135 @@ type state struct {
 	autoescape   ast.AutoescapeType
 	lastNode     ast.Node
 	options      Options
+	funcsCalled  map[string]string
+	funcsInFile  map[string]bool
+}
+
+// ES5Formatter implements the JSFormatter interface
+// and creates Javascript files following the ES5
+// Javascript format (without imports)
+type ES5Formatter struct{}
```

Comment:
I think the comment got lost, but could you break these types out into a formatters.go file?